### DISCUSSION TITLE
Modal Body doesn't scroll as expected

### DISCUSSION THREAD

**Discussion by chjpark** [twilio-labs/paste#1337] [Q&A] [answered]
Hi,

I'm using a modal and I have two TextArea paste components in the ModalBody. When the TextArea increases in height as more text gets entered into it, I see it push the FooterAction buttons out of the ModalFooter. According to the docs for the ModalBody:

"A Modal has a default width of 608 px to allow for an optimal reading length at 70-75 characters per line at default Paragraph size at 100% magnification. At viewports smaller than 608 px, the Modal will fill the width of the viewport and pin to the bottom. The Modal will grow in height with the amount of content in it. Once the modal reaches 90% of the height of the viewport, the body will begin to scroll."

But I've reached 90% of the height of the viewport and the ModalBody doesn't scroll but pushes the ModalFooterAction buttons out of the modal. In the example below, I have the error help text show up and that pushes the Action Buttons out.

<img width="897" alt="Screen Shot 2021-04-05 at 1 47 34 PM" src="https://user-images.githubusercontent.com/1145733/113625465-d2cc0e00-9615-11eb-9d15-3ff90cfdca5e.png">

Is there a property that I am missing to prevent the modal buttons in the ModalFooter from being pushed out by heigh changes within the ModalBody?

**Comment by serifluous** [comment, marked as answer]
Hey @chjpark ! @richbachman did some digging, and we couldn't reproduce the issue ourselves in CodeSandbox: https://codesandbox.io/s/textarea-modal-example-s73mf?file=/src/index.tsx

Can you see if our example helps out? If it doesn't, could you create your own CodeSandbox to help us reproduce the issue, or come to office hours tomorrow and we can help you out live?

**Reply by chjpark** [reply]
Hey, I'd like to stop by during the office hours to help reproduce the issue. What time is the office hours?

**Reply by corinne-nagel** [reply]
Hi @chjpark , apologies on this late response! Our office hours are tomorrow, here's [the invite](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=800758837) (the slots say closed but if you can make it we can add you since we dropped the ball on this convo).

**Reply by gloriliale** [reply]
Hi @chjpark! ~~Please feel free to [sign up for an office hours slot here](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049) (they're very open right now but can fill up quickly) so that we can figure this out!~~ 
apologies for the miscommunication, here's the discussion during office hours for reference, starting at 20:30!
https://www.loom.com/share/45266766429f4455b99e94e6c7af1cca
